### DISCUSSION TITLE
win key shift s (snipping tool) no longer works

### DISCUSSION THREAD

**Discussion by wpg1993** [valinet/ExplorerPatcher#723] [General]
the key bind no longer works and snipping tool just captures itself 
print screen still works  but not snipping
not sure if its a bug for me or just an issue everyone has

**Comment by chip33** [comment]
Hi, just tested it and its working for me on the latest pre-release of EP and the latest Stable windows 11 release.

**Reply by wpg1993** [reply]
![Screenshot 2022-01-24 213336](https://user-images.githubusercontent.com/78664594/150900427-87918a95-cd3c-4169-b3b4-95cf27c79fa0.png)
this is what happens when i try 
it just crashes or the key bind doesn't work
closing everything opens explore 
is it because i have this and open shell (classic shell) running together 
i even hit the update via properties and tried the latest build on here
broth give me the exact same issue
